molecule/scram-rhel/verify.yml

```diff
@@ -14,7 +34,43 @@
       vars:
         file_path: /etc/kafka/server.properties
         property: listener.name.internal.sasl.enabled.mechanisms
+        expected_value: SCRAM-SHA-512,SCRAM-SHA-256
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/kafka/server.properties
+        property: sasl.mechanism.controller.protocol
+        expected_value: GSSAPI
+      when: kraft_mode|bool
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_property.yml
+      vars:
+        file_path: /etc/kafka/server.properties
+        property: sasl.mechanism.controller.protocol
```

Comment:
why will this config be present in non kraft setup?

Reply:
Thanks for pointing out, missed it.